src/test/tests/databases/xdmf.py

```diff
@@ -136,7 +136,7 @@ def test2(datapath):
     m.opaqueMode = m.Auto  # Auto, On, Off
     m.pointSize = 0.05
     m.opaqueColor = (192, 192, 192, 255)
-    m.smoothingLevel = m.None  # None, Fast, High
+    m.smoothingLevel = m.NONE  # None, Fast, High
```

Comment:
Am confused here regarding `None` vs `NONE`. I guess in Python 3 `None` is a keyword now? And, the solution was to upcase VisIt's symbol. But that now means we have this one enum in VisIt with odd-ball case everywhere. I know this is a simple style issue and so don't think we should worry about adjusting here. But, I think a sep. ticket to replace VisIt's now `NONE` with a suitable synonym such as `Nothing`, `Nil`, `Void`, etc.

Reply:
That is correct, we cant use `None` any more. 

We can discuss more if you like (please add a ticket) -- but most use cases involve people getting these values through recording or printing. The old lowercase names are still supported (just not in python 3)

Reply:
Added #4920

Reply:
thanks, you captured related thoughts well!